packages/flutter/lib/src/widgets/scroll_physics.dart

```diff
@@ -195,6 +195,18 @@ class ScrollPhysics {
   /// Scroll fling velocity magnitudes will be clamped to this value.
   double get maxFlingVelocity => parent?.maxFlingVelocity ?? kMaxFlingVelocity;
 
+  /// Calculate velocity carried on repeated flings.
```

Comment:
nit: slightly more precise `/// Returns the velocity carried on repeated flings.`

(also fixes the pedantic 'start method docs with a third-person verb' nit)

Reply:
Done